### PR TITLE
chore(config) default api_url and grpc_url in the app

### PR DESCRIPTION
**Pull request by mtmr0x**

### CODE REVIEW

File: gateway/appconfig/appconfig.go [hunk 1]
```diff
@@ -57,9 +58,14 @@ func Load() error {
...
 	if apiURL == "" {
-		return fmt.Errorf("API_URL is required but is empty")
+		apiURL = "http://127.0.0.1:8009"
+	}
+	grpcURL := os.Getenv("GRPC_URL")
+	if grpcURL == "" {
+		grpcURL = "http://127.0.0.1:8010"
 	}
 	apiRawURL, err := url.Parse(apiURL)
+	fmt.Printf("apiRawURL: %+v\n", apiRawURL.Host)
```
Comment (sandromello) [inline gateway/appconfig/appconfig.go]: Remove this please

File: gateway/main.go [hunk 2]
```diff
@@ -64,7 +64,7 @@ func Run() {
 	}
 
 	idProvider := idp.NewProvider(apiURL)
-	grpcURL := os.Getenv("GRPC_URL")
+	grpcURL := appconfig.Get().GrpcURL()
 	if grpcURL == "" {
 		scheme := "grpcs"
 		if appconfig.Get().ApiScheme() == "http" {
```
Comment (sandromello) [inline gateway/main.go]: Remove this condition, it will never be triggered

Comment (mtmr0x) [reply gateway/main.go]: you mean the whole condition, beginning at line 68, right?

Comment (sandromello) [reply gateway/main.go]: Yes

**Review by sandromello — APPROVED** [review]

**Pull request merged**